operator/apis/lifecycle/v1alpha2/keptnevaluation_test.go

```diff
@@ -15,9 +15,10 @@ func TestKeptnEvaluation(t *testing.T) {
 			Name: "evaluation",
 		},
 		Spec: KeptnEvaluationSpec{
-			AppName:    "app",
-			AppVersion: "appversion",
-			Type:       common.PostDeploymentCheckType,
+			AppName:              "app",
```

Comment:
These values are used in multiple places of the test. It would be better to have the values as constant, so there are fewer chances of errors due to typos, and you have a single place where to update them.

Reply:
This also applies to other tests. We can refactor this in a follow-up if you want

Reply:
I would say this will need another refactoring round and would rather not put it in this PR